### DISCUSSION TITLE
Does it cause any problem sending queries from the client to the backend? [security etc]

### DISCUSSION THREAD

**Discussion by aramis-it** [prisma/prisma#4819] [Q&A]
For example

```
fetch('url',{
  where: { id: 1 },
  data: { email: 'alice@prisma.io' },
})
```

**Comment by nikolasburk** [comment]
Hey @ulziibadrakh-p, I'm not sure I fully understand your question, would you mind elaborating a bit on what exactly you are trying to achieve? 

Your code snippet seems like you're sending an HTTP request using `fetch` to an API. As parameters you're passing an object that looks like it could be passed to an `update` function of Prisma Client. I don't think that there is generally anything wrong with this setup.
